jsHelper/spicetifyWrapper.js

```diff
@@ -303,6 +303,90 @@ window.Spicetify = {
 	Platform: {}
 };
 
+(function addProxyCosmos() {
+	if (!Spicetify.Player.origin?._cosmos) {
+		setTimeout(addProxyCosmos, 50);
+		return;
+	}
+
+	const corsProxyURL = "https://cors-proxy.spicetify.app/";
+	const allowedMethodsMap = {
+		get: "get",
+		post: "post",
+		del: "delete",
+		put: "put",
+		patch: "patch"
+	};
+	const allowedMethodsSet = new Set(Object.keys(allowedMethodsMap));
+	const internalEndpoints = new Set(["sp:", "wg:"]);
+
+	const handler = {
+		// biome-ignore lint/complexity/useArrowFunction: <explanation>
+		get: function (target, prop, receiver) {
+			const internalFetch = Reflect.get(target, prop, receiver);
+
+			if (typeof internalFetch !== "function" || !allowedMethodsSet.has(prop) || Spicetify.Platform.version < "1.2.31") return internalFetch;
+
+			// biome-ignore lint/complexity/useArrowFunction: <explanation>
```

Comment:
Can't we not have `<explanation>`? That's alright if not, but we should briefly explain then, right?